tests/integration/ceph_mgr_test.go

```diff
@@ -205,6 +206,31 @@ func (s *CephMgrSuite) waitForOrchestrationModule() {
 	}
 	require.Nil(s.T(), err)
 }
+
+func (s *CephMgrSuite) CompareNodes(orchestratorNodes []string) error {
+	// get the k8s nodes
+	nodes, err := k8sutil.GetNodeHostNames(s.k8sh.Clientset)
+	if err == nil {
+		if len(nodes) == 0 {
+			return fmt.Errorf("k8s cluster does not return any cluster nodes")
+		} else {
```

Comment:
no need for the `else` block since the previous `if` returned
```suggestion
		} 
```

Reply:
If we reach this point we have executed the orchestrator command successfully, but the  output could be an empty string ... so it is needed to process what happens depending of the output returned .
I thing that we cannot eliminate the "else" part.